client/java/client-java-configurations.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Java Client 配置参数
```

Comment:
Please integrate this doc into the existing TiDB doc system instead of having it isolated from other docs.

Reply:
@scsldb @kissmydb Please also take a look, thanks!

Reply:
I have not found any document in this repo about clients yet (client-go, client-c, client-rust, client-java). If there is a better place to integrate, I would be happy to move the documents there.

Reply:
Suggest placing the directory and file name as: `client/client-java-configuration.md` and `/client/use-client-java.md `, because the PingCAP doc site can only parse two-layers directory. 

For me, `/client` is OK. @kissmydb What do you think?

Reply:
Suggest adding the summary after the title.

Reply:
为了确认文档的目录位置，请问：在 TiDB 的用户旅程中，什么时候会用到 Java client？ @birdstorm